dev-tools/mage/build.go

```diff
@@ -91,7 +91,7 @@ func GolangCrossBuild(params BuildArgs) error {
 			"only be executed within the golang-crossbuild docker environment.")
 	}
 
-	defer DockerChown(filepath.Join(params.OutputDir, params.Name+binaryExtension(GOOS)))
+	defer DockerChown(params.OutputDir)
```

Comment:
How come this was changed? Won't this be `chown`ing files unnecessarily?

Reply:
I've changed the logic of DockerChown to not recursively walk up the path, instead it walk down the path, using only `params.outputDir` will change the permission bellow that directory instead. The other implementation had issues with using absolute paths.

Reply:
I will revert it back, it will work with both logic.